### PR TITLE
Keyword `default`

### PR DESCRIPTION
**Pull request by AgniveshChaubey**
This PR addresses #73.

### CODE REVIEW

File: content/2020-12/meta-data/default.markdown [hunk 1]
```diff
@@ -26,3 +26,138 @@ Annotations
 -----------
 
 This keyword produces the default value as the annotation value.
+
+## Explanation
+
+The `default` keyword in JSON Schema is used to specify a default value for an instance or its parts. This value is not used to fill in missing values during the validation process but can be used by non-validation tools such as documentation generators or form generators to provide hints to users.
```
Comment (jviotti) [inline content/2020-12/meta-data/default.markdown:32]:
```suggestion
The `default` keyword in JSON Schema is used to specify a default value for an instance. This value is not automatically used to fill in missing values during the validation process but can be used by tools such as documentation or form generators.
```

File: content/2020-12/meta-data/default.markdown [hunk 2]
```diff
@@ -26,3 +26,138 @@ Annotations
...
+
+_**Note:** While it is recommended that the default value validate against its subschema, this requirement is not strictly enforced. However, it is strongly advised for people to ensure validation compatibility_
```
Comment (jviotti) [inline content/2020-12/meta-data/default.markdown:34]:
```suggestion
_**Note:** While it is recommended that the default value validate against its subschema, this requirement is not strictly enforced. 
```

I think the second sentence just says the same thing

File: content/2020-12/meta-data/default.markdown [hunk 3]
```diff
@@ -26,3 +26,138 @@ Annotations
...
+    }
+  },
+  "if": {
+    "properties": {
+      "education": { "const": "degree" }
+    }
+  },
+  "then": {
+    "properties": {
+      "degree_certificate": {
```
Comment (jviotti) [inline content/2020-12/meta-data/default.markdown:76]: Can we use camelCase, as that's the style JSON Schema encourages?

File: content/2020-12/meta-data/default.markdown [hunk 4]
```diff
@@ -26,3 +26,138 @@ Annotations
...
+  "if": {
+    "properties": {
+      "qualification": { "const": "degree" }
+    }
+  },
+  "then": {
+    "properties": {
+      "degreeCertificate": {
+        "type": "string",
+        "default": "XXXXXX"
```
Comment (jviotti) [inline content/2020-12/meta-data/default.markdown:78]: These placeholders feel a bit odd. Maybe generate a random short alphanumeric string instead? Something like `B0B8RKEZ90`? Would feel at least a bit more plausible

File: content/2020-12/meta-data/default.markdown [hunk 5]
```diff
@@ -26,3 +26,138 @@ Annotations
...
+      }
+    },
+    "required": [ "degreeCertificate" ]
+  },
+  "else": {
+    "properties": {
+      "diplomaCertificate": {
+        "type": "string",
+        "default": "YYYYYY"
```
Comment (jviotti) [inline content/2020-12/meta-data/default.markdown:87]: Same here. Maybe `PW458C468E`?

File: content/2020-12/meta-data/default.markdown [hunk 6]
```diff
@@ -26,3 +26,138 @@ Annotations
...
+    "required": [ "diplomaCertificate" ]
+  }
+}
+{{</schema>}}
+
+{{<instance-pass `An instance conforming to the schema is valid`>}}
+{
+  "name": "Doe",
+  "qualification": "degree",
+  "degreeCertificate": "XXYYZZ"
```
Comment (jviotti) [inline content/2020-12/meta-data/default.markdown:99]: `O5CYPZACTN`?

**Review by jviotti — APPROVED** [review]
I left a few comments regarding the placeholders, but looks good given that

File: content/2020-12/meta-data/default.markdown (hunk 4 above)
Comment (AgniveshChaubey) [reply content/2020-12/meta-data/default.markdown:78]: Make sense!

**Pull request merged**